pkg/aggregator/ckey/key_test.go

```diff
@@ -49,21 +51,100 @@ func TestCompare(t *testing.T) {
 	assert.False(t, Equals(base, diff))
 }
 
-func genTags(count int) []string {
+func TestTagsOrderAndDupsDontMatter(t *testing.T) {
```

Comment:
In [this experiment](https://github.com/djmitche/tagset-experiment/blob/e4085f2250a4357338e5d8cb0f49375d0f30f1d9/tagset/tagset_test.go#L92-L138) I did some seeded randomized trials as a kind of lightweight fuzzing.  Maybe worth adding a similar thing here?